ert_storage/database_schema.py

```diff
@@ -78,6 +91,11 @@ def __init__(
     f64_matrix = relationship("F64Matrix")
     ensemble_id = sa.Column(sa.Integer, sa.ForeignKey("ensemble.id"), nullable=True)
     ensemble = relationship("Ensemble", back_populates="records")
+    observations = relationship(
+        "Observation",
+        secondary=observation_record_association,
```

Comment:
Don't fully understand this `observation_record_association` (it might best just me though). Is this like when record is a response, this provides like to the observation?

Reply:
the observation_record_association, is creating a table for having a many-to-many relation in the db.